src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java

```diff
@@ -84,244 +77,147 @@
   private static Thread rcaNetServerThread = null;
 
   public static final SampleAggregator RCA_GRAPH_METRICS_AGGREGATOR =
-          new SampleAggregator(RcaGraphMetrics.values());
-  public  static final SampleAggregator RCA_RUNTIME_METRICS_AGGREGATOR =
-          new SampleAggregator(RcaRuntimeMetrics.values());
+      new SampleAggregator(RcaGraphMetrics.values());
```

Comment:
Looks like it was previously not (strictly) formatted according to the google style guide. Now it adheres to the style guide correctly which is enforced by the formatter in intellij. The continuation indent parameter which controls this is defined as follows:
> "When line-wrapping, each line after the first (each continuation line) is indented at least +4 from the original line. When there are multiple continuation lines, indentation may be varied beyond +4 as desired."

instead of varying the space beyond +4, it is strictly setting it to exactly +4 now.